integrations/oci/oci-secrets-config-source/src/main/java/io/helidon/integrations/oci/secrets/configsource/OciSecretsConfigSourceProvider.java

```diff
@@ -0,0 +1,526 @@
+                        if (re == null) {
+                            re = e;
+                        } else {
+                            re.addSuppressed(e);
+                        }
+                    } catch (Exception e) {
+                        if (e instanceof InterruptedException) {
+                            Thread.currentThread().interrupt();
+                        }
+                        if (re == null) {
+                            re = new IllegalStateException(e.getMessage(), e);
+                        } else {
+                            re.addSuppressed(e);
+                        }
+                    }
+                }
+            } catch (RuntimeException e) {
+                re = e;
+            } catch (Exception e) {
+                if (e instanceof InterruptedException) {
+                    Thread.currentThread().interrupt();
+                }
+                re = new IllegalStateException(e.getMessage(), e);
+            } finally {
+                try {
+                    secrets.close();
+                } catch (RuntimeException e) {
+                    if (re == null) {
+                        throw e;
+                    } else {
+                        re.addSuppressed(e);
+                    }
+                } catch (Exception e) {
+                    if (e instanceof InterruptedException) {
+                        Thread.currentThread().interrupt();
+                    }
+                    if (re == null) {
+                        throw new IllegalStateException(e.getMessage(), e);
+                    } else {
+                        re.addSuppressed(e);
+                    }
+                }
+                if (re != null) {
+                    throw re;
+                }
+            }
+        }
+
+        @SuppressWarnings("try")
+        private static Collection<? extends SecretSummary> secretSummaries(Supplier<? extends Vaults> vaultsSupplier,
+                                                                           ListSecretsRequest listSecretsRequest) {
+            try (Vaults v = vaultsSupplier.get()) {
+                return v.listSecrets(listSecretsRequest).getItems();
+            } catch (RuntimeException e) {
+                throw e;
+            } catch (Exception e) {
+                if (e instanceof InterruptedException) {
+                    Thread.currentThread().interrupt();
+                }
+                throw new IllegalStateException(e.getMessage(), e);
+            }
+        }
+
+        @SuppressWarnings("checkstyle:linelength")
+        private static ValueNode valueNode(Secrets s, SecretSummary ss, Base64.Decoder d) {
+            return
+                ValueNode.create(new String(d.decode(((Base64SecretBundleContentDetails) (s.getSecretBundle(GetSecretBundleRequest.builder()
+                                                                                                                .secretId(ss.getId())
```

Comment:
this is really hard to read due to spacing and compounding of actions - please consider breaking it up and realigning it.

Reply:
Done